pandas/core/series.py

```diff
@@ -3800,9 +3800,17 @@ def _gotitem(self, key, ndim, subset=None):
         **_shared_doc_kwargs
     )
     @Appender(generic._shared_docs["aggregate"])
-    def aggregate(self, func, axis=0, *args, **kwargs):
+    def aggregate(self, func=None, axis=0, *args, **kwargs):
         # Validate the axis parameter
         self._get_axis_number(axis)
+
+        if func is None:
+            # This is due to order issue of dictionary in PY35, e.g. if {"foo"
```

Comment:
py35 has been dropped; can this be simplified?

Reply:
yes, `OrderDict` is no longer needed